test/powershell/engine/Basic/SemanticVersion.Tests.ps1

```diff
@@ -188,7 +196,6 @@ Describe "SemanticVersion api tests" -Tags 'CI' {
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.+alpha" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0-alpha+" }
             @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0-+" }
-            @{ name = "Missing parts";               errorId = "FormatException";    expectedResult = $false; version = "1.0.0+-" }
```

Comment:
Can you add a positive test with the expected results?

Reply:
It's been added.